stubs/pygit2/pygit2/enums.pyi

```diff
@@ -1,395 +1,395 @@
 from enum import IntEnum, IntFlag
```

Comment:
I tried to set all enum values in this module to `...` but stubtest was mad about it so I generated all of them with a script.